PlayerControls/sources/DefaultControlsViewController.swift

```diff
@@ -179,6 +179,12 @@ public final class DefaultControlsViewController: ContentControlsViewController
             
             resetUrlIfNeeded()
             handleThumbnail()
+        } else if let image = uiProps.thumbnailImage {
+            func handleThumbnailImage() {
```

Comment:
What is the purpose of this function if it is called immediately?

Reply:
Will be removed in the next commit.